tools/watch.js

```diff
@@ -267,7 +267,6 @@ var readDirectory = function (options) {
       }
       throw e;
     }
-    // XXX if we're on windows, I guess it's possible for files to end with '/'.
```

Comment:
I deleted an old comment that was concerned about the future windows development. I think the comment is not relevant any more, so I deleted it.